service/src/main/java/io/camunda/service/ManagementServices.java

```diff
@@ -25,4 +25,12 @@ public boolean isCamundaLicenseValid() {
   public LicenseType getCamundaLicenseType() {
     return license.getLicenseType();
   }
+
+  public boolean isCommercialCamundaLicense() {
+    return license.isCommercial();
+  }
+
+  public String getCamundaLicenseExpirationDate() {
+    return license.expiresAt();
+  }
```

Comment:
🔧 Let the consumers returning data care about the format, not the service layer. That way, consumers have the opportunity to provide dates consistently. Otherwise, they can either only ship formatted date strings as received (bad UX) or parse and reformat (horrible implementation).